pkgs/applications/misc/emem/default.nix

```diff
@@ -2,14 +2,14 @@
 
 stdenv.mkDerivation rec {
   pname = "emem";
-  version = "0.2.41";
+  version = "0.2.42";
   name = "${pname}-${version}";
 
   inherit jdk;
 
   src = fetchurl {
```

Comment:
yup, easily. you look at the release url, such as https://github.com/ebzzry/emem/archive/v0.2.42.tar.gz at the end it is v0.2.42 ; so your rev would be:
rev = "v${version}";